### PR TITLE
docs(material/icon): add harness example for icon

### PR DESCRIPTION
**Pull request by annieyw**

### CODE REVIEW

**Review by crisbeto — APPROVED** [review]
LGTM, can be marked as ready when the lint failure is fixed.

File: src/components-examples/material/icon/icon-harness/icon-harness-example.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        declarations: [IconHarnessExample]
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    const registry = TestBed.inject(MatIconRegistry);
+    const sanitizer = TestBed.inject(DomSanitizer);
+
+    registry.addSvgIconLiteralInNamespace('svgIcons', 'svgIcon',
```
Comment (jelbourn) [inline src/components-examples/material/icon/icon-harness/icon-harness-example.spec.ts:33]: I would add a comment here like:
```ts
// We use `bypassSecurityTrustHtml` exclusively for testing here.
```

**Review by jelbourn — APPROVED** [review]
LGTM

File: src/components-examples/material/icon/icon-harness/icon-harness-example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import {HarnessLoader, parallel} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {IconHarnessExample} from './icon-harness-example';
+import {MatIconModule, MatIconRegistry} from '@angular/material/icon';
+import {MatIconHarness} from '@angular/material/icon/testing';
+import {DomSanitizer} from '@angular/platform-browser';
+
+
+describe('TableHarnessExample', () => {
```
Comment (crisbeto) [inline src/components-examples/material/icon/icon-harness/icon-harness-example.spec.ts:12]: This should be renamed.

File: src/material-experimental/mdc-form-field/_form-field-theme.scss [hunk 3]
```diff
@@ -15,7 +15,7 @@
 // the given theme palette. The MDC text-field is styled using `primary` by default.
 @mixin _mat-mdc-text-field-color-styles($palette-name, $query: $mat-theme-styles-query) {
   $_mdc-text-field-focused-label-color: $mdc-text-field-focused-label-color;
-  $mdc-text-field-focused-label-color: rgba(mdc-theme-prop-value($palette-name), 0.87) !global;
+  $mdc-text-field-focused-label-color: rgba(mdc-theme-prop-value($palette-name), 1) !global;
```
Comment (crisbeto) [inline src/material-experimental/mdc-form-field/_form-field-theme.scss:18]: This change looks unrelated.

**Review by crisbeto — APPROVED** [review]
LGTM, but removing the "merge ready" label due to the two comments above. Can be set to "merge ready" again once they've been resolved.
